packages/terra-application/CHANGELOG.md

```diff
@@ -5,7 +5,7 @@
 * Changed
   * Updated jest snapshots for terra-icon and terra-button changes.
   * Updated size explanations for ModalManager managed by DisclosureManagerContext.
-  * Updated `uuid` to `v8` for consistency with other components.
+  * Updated `uuid` to `v8.2.0` for consistency with other components.
```

Comment:
nit:
```suggestion
  * Updated `uuid` to `8.2.0` for consistency with other components.
```

Reply:
Fixed in [this commit](https://github.com/cerner/terra-application/pull/335/commits/d4ca51e2dff00709bf151abde4b543c4b62b8296)